projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -4907,6 +4907,160 @@ public string FatherEducationLevelEditFlagHelper
+        public string BirthFacilityName
+        {
+            get => GetFacilityLocation(ValueSets.LocationTypes.Birth_Location)?.Name;
+            set => (GetFacilityLocation(ValueSets.LocationTypes.Birth_Location) ?? CreateAndSetLocationBirth(ValueSets.LocationTypes.Birth_Location)).Name = value;
+        }
+
+        /// <summary>Name of Facility mother moved from (if transfered)</summary>
+        /// <value>FacilityMotherTransferredFrom.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.FacilityMotherTransferredFrom = "South Hospital";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Facility Mother Transferred From: {ExampleBirthRecord.FacilityMotherTransferredFrom}");</para>
+        /// </example>
+        [Property("Facility Mother Transferred From", Property.Types.String, "Birth Location", "Facility Mother Moved From (if transferred)", true, IGURL.LocationBFDR, true, 34)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='87300-0')", "")]
+        public string FacilityMotherTransferredFrom
+        {
+            get => GetFacilityLocation(ValueSets.LocationTypes.Transfer_From_Location)?.Name;
+            set => (GetFacilityLocation(ValueSets.LocationTypes.Transfer_From_Location) ?? CreateAndSetLocationBirth(ValueSets.LocationTypes.Transfer_From_Location)).Name = value;
+        }
+
+        /// <summary>Name of Facility Infant Transferred To (if transfered w/in 24 hours)</summary>
+        /// <value>FacilityInfantTransferredTo.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.FacilityInfantTransferredTo = "South Hospital";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Facility Mother Transferred From: {ExampleBirthRecord.FacilityInfantTransferredTo}");</para>
+        /// </example>
+        [Property("cility Infant Transferred To", Property.Types.String, "Birth Location", "Facility Infant Transferred To (if transferred w/in 24 hours)", true, IGURL.LocationBFDR, true, 34)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='87300-0')", "")]
+        public string FacilityInfantTransferredTo
+        {
+            get => GetFacilityLocation(ValueSets.LocationTypes.Transfer_To_Location)?.Name;
+            set => (GetFacilityLocation(ValueSets.LocationTypes.Transfer_To_Location) ?? CreateAndSetLocationBirth(ValueSets.LocationTypes.Transfer_To_Location)).Name = value;
+        }
+
+        private Location GetFacilityLocation(string code) {
+            return (Location)Bundle.Entry.Where(e => e.Resource is Location loc && loc.Type.Any(type => type.Coding.Any(coding => coding.System == CodeSystems.LocalBFDRCodes && coding.Code == code))).FirstOrDefault()?.Resource;
+        }
+        
+        private Observation GetObservation(string code)
```

Comment:
This helper was refactored into VitalRecord_util in a previous merged PR. There should be a function in there called GetObservation that replaced this one. This one can be removed.

Reply:
Resolved.